hack/check-error-log-msg-format.sh

```diff
@@ -6,7 +6,7 @@ set -o pipefail
 source "hack/lib/test_lib.sh"
 
 echo "Checking format of error and log messages..."
-allfiles=$(listFiles)
+allfiles=$(listValidFilesForSanityCheck)
```

Comment:
Instead of changing these functions, can you just do
```suggestion
allfiles=$(listFiles | grep -v _bindata)
```
then we can preserve these functions for other use and only exclude generated bindata files, allowing these checks to run on manually written files.